blocks/carousel/carousel.css

```diff
@@ -0,0 +1,94 @@
+.carousel {
+  width: 100%;
+  margin-bottom: 2rem;
+  position: relative;
+  display: flex;
+  justify-content: center;
+}
+  
+
+.carousel>ul {
+  width: 70%;
+  display: flex;
+  flex-wrap: nowrap;
+  overflow-x: scroll;
+  gap: 2rem;
+  list-style: none;
+  scroll-behavior: smooth;
+  scroll-snap-type: x mandatory;
+}
+
+.carousel>ul>li {
+  scroll-snap-align: start;
+  
```

Comment:
Something like this to display 1 card only by default:
flex: 1;
min-wdith: 100%;